src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/LoadProfilerActionTests.cs

```diff
@@ -0,0 +1,92 @@
+
+using Microsoft.Diagnostics.Monitoring.TestCommon.Options;
+using Microsoft.Diagnostics.Monitoring.Tool.UnitTests.CollectionRules.Actions;
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Actions;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    public sealed class LoadProfilerActionTests
+    {
+        private readonly ITestOutputHelper _outputHelper;
+        private static readonly TimeSpan TimeoutMs = TimeSpan.FromSeconds(1);
+        private const string DefaultRuleName = nameof(ActionDependencyAnalyzerTests);
+
+        public LoadProfilerActionTests(ITestOutputHelper outputHelper)
+        {
+            _outputHelper = outputHelper;
+        }
+
+        [Fact]
+        public async Task TranslateSingleStringToAny()
+        {
+            const string TargetPath = @"C:\My\Test\Path";
+            LogRecord record = new LogRecord();
+            LoadProfilerOptions settings = new LoadProfilerOptions()
+            {
+                Path = TargetPath,
+                ProfilerGuid = Guid.NewGuid(),                    
+            };
+            await TestHostHelper.CreateCollectionRulesHost(_outputHelper, rootOptions =>
+            {
+                CollectionRuleOptions options = rootOptions.CreateCollectionRule(DefaultRuleName)
+                    .AddAction(KnownCollectionRuleActions.LoadProfiler, settings, out CollectionRuleActionOptions actionOptions)
+                    .SetStartupTrigger();
+
+                settings = (LoadProfilerOptions)options.Actions.Last().Settings;
+            }, host =>
+            {
+                using CancellationTokenSource cancellationTokenSource = new CancellationTokenSource(TimeoutMs);
+
+                IOptionsMonitor<CollectionRuleOptions> collectionRuleOptsMonitor = host.Services.GetRequiredService<IOptionsMonitor<CollectionRuleOptions>>();
+                CollectionRuleOptions ruleOptions = collectionRuleOptsMonitor.Get(DefaultRuleName);
+                ILogger<CollectionRuleService> logger = host.Services.GetRequiredService<ILogger<CollectionRuleService>>();
+
+                CollectionRuleContext context = new(DefaultRuleName, ruleOptions, null, logger);
+
+                LoadProfilerOptions configuredOptions = context.Options.Actions.First().Settings as LoadProfilerOptions;
+                Assert.NotNull(configuredOptions);
+                Assert.True(configuredOptions.PathMap.ContainsKey("any"));
```

Comment:
It looks like we have a bunch of references to the string literal `"any"`; maybe toss this into a constant string like you did in `RidResolver`?